ingestion/setup.py

```diff
@@ -370,7 +370,7 @@
     VERSIONS["grpc-tools"],
     VERSIONS["neo4j"],
     "testcontainers==3.7.1;python_version<'3.9'",
-    "testcontainers==4.8.0;python_version>='3.9'",
+    "testcontainers==4.8.1;python_version>='3.9'",
```

Comment:
Should we just fix it on minor version so we keep adding patches release? 

```suggestion
    "testcontainers~=4.8.0;python_version>='3.9'",
```

Reply:
We can do that later

Reply:
done

Reply:
didnt make the commit... I will file a separate PR for this.

Reply:
https://github.com/open-metadata/OpenMetadata/pull/17938